src/main/java/org/embulk/filter/column/ColumnFilterPlugin.java

```diff
@@ -117,7 +120,7 @@ static Schema buildOutputSchema(PluginTask task, Schema inputSchema)
                 boolean matched = false;
                 for (ColumnConfig dropColumn : dropColumns) {
                     // skip json path notation to build outputSchema
-                    if (dropColumn.getName().startsWith("$.")) {
+                    if (PathCompiler.isDocContext(dropColumn.getName())) {
```

Comment:
This is originally constants it named `DOC_CONTEXT`.
But this method name is difficult to understand as you say.

I would like to change the method name `isStartsWithDoller`.